### DISCUSSION TITLE
Copilot automatically disabled in WSL2, says "Copilot is disabled because we could not fetch the repository policy"

### DISCUSSION THREAD

**Discussion by lazydreamerbliss** [community/community#101988] [Copilot Conversations] [answered]
### Select Topic Area

Copilot

### Body

I use vscode to open my project in WSL2 from windows host (run `code .` command in terminal from windows under WSL2 directory), and every time the Copilot will be disabled says "Copilot is disabled because we could not fetch the repository policy", and here is the log:

```
[INFO] [CopilotContentExclusion] [2024-01-25T09:54:17.668Z] feature flag is enabled for user
[INFO] [CopilotContentExclusion] [2024-01-25T09:54:18.687Z] contentExclusion.evaluate, No matching policy for this repository. uri: /home/path/to/project
[INFO] [CopilotContentExclusion] [2024-01-25T09:54:18.688Z] repositoryControl.evaluate,/home/path/to/project,{"isBlocked":true,"reason":"POLICY_ERROR","message":"Copilot is disabled because we could not fetch the repository policy"}
```

This error only occurs when I open a project from WSL, no matter the project is on WSL directory or windows directory. Do I need to config something to make Copilot work? Thanks in advance!

**Comment by inksnw** [comment]
I have the same issue

**Comment by REIGN12** [comment]
I am having the same error message, though not in the WSL. I am connecting a docker container using vscode. Why this is happening...

**Reply by REIGN12** [reply]
And this is not a single cases, some of my friends also encounter this issue...

**Comment by xinsnake** [comment]
I am having this issue in a pure environment on a physical computer. Having this problem on both VSCode and JetBrains.

**Reply by marrobi** [reply]
I've hit this today too. Wonder if its a service issue. VSCode and a DevContainer on WSL 2.

**Comment by shubham1172** [comment]
I am having this issue on VS Code and VS Code-Insiders on M1 as well from today. Tried installing previous versions of Copilot too, does not help.

**Comment by pedromartinssouza** [comment]
Same thing here. Pure Windows, getting this error in VSCode.

**Comment by Toudsour** [comment]
I am having this issue in my Linux computer.

**Comment by oasaleh** [comment]
I have this error on both VS2022 and VSCode.

**Comment by asalcedo29** [comment]
Having this same error in VSCode.

**Comment by NicolasOe** [comment]
Having the same issue in VS Code

**Comment by VladyslavNesterovskyi** [comment]
Looks like some WW issue - same problem.

**Comment by hsenag** [comment, marked as answer]
Hi, there's a known problem - now resolved - which was specifically affecting Microsoft users. Feel free to ping me (`asittampalam`) and @monil-patel (`mopatel`) internally for more details about that.

If you're not from Microsoft please contact GitHub Support if you're having this problem

**Reply by hassanin** [reply]
The issue seems to have been resolved now

**Reply by lazydreamerbliss** [reply]
Hi hsenag, the issue is still there for me, I have the latest VSCode version with updated copilot extension, but the issue is still there when I open a WSL2-hosted project, do you have ideas to it? Thanks!

**Reply by hsenag** [reply]
@lazydreamerbliss sounds like you have some more specific problem. Please ping me internally if you haven't already.

**Reply by nuruddinshohan83** [reply]
[info] [auth] Invalid copilot token: missing token: 403 
[error] [auth] Extension activation failed: "Contact Support. You are currently logged in as nuruddinshohan83."
@hsenag  I'm getting this error on my wsl2 vscode

**Reply by hsenag** [reply]
@nuruddinshohan83 That's a different error to the one in this thread. You need to contact support as the message says.

**Reply by nuruddinshohan83** [reply]
thanks for your response

**Reply by Mydurian** [reply]
Hi @hsenag  I have encountered the same issue. 
statusNotification: Inactive: Copilot is disabled because we could not fetch the repository policy
